### PR TITLE
internal: update chakracore port

### PR DESCRIPTION
**Pull request by Pospelove**
see https://github.com/microsoft/vcpkg/pull/28472

### CODE REVIEW

File: overlay_ports/chakracore/add-missing-reference.patch [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+index 1b13cea..57b066b 100644
+--- a/lib/SCACore/Chakra.SCACore.vcxproj
++++ b/lib/SCACore/Chakra.SCACore.vcxproj
+@@ -60,6 +60,9 @@
+     <ClInclude Include="StreamReader.h" />
+     <ClInclude Include="StreamWriter.h" />
+   </ItemGroup>
++  <ItemGroup>
++    <ProjectReference Include="$(MSBuildThisFileDirectory)../../manifests/CoreManifests.vcxproj" />
++  </ItemGroup>
```
Comment (nic11) [inline overlay_ports/chakracore/add-missing-reference.patch:11]: What is this doing?

File: overlay_ports/chakracore/portfile.cmake [hunk 2]
```diff
@@ -1,144 +1,132 @@
-vcpkg_check_linkage(ONLY_DYNAMIC_LIBRARY)
```
Comment (nic11) [inline overlay_ports/chakracore/portfile.cmake:1]: Run `dos2unix overlay_ports/chakracore/portfile.cmake` please

File: overlay_ports/chakracore/add-missing-reference.patch (hunk 1 above)
Comment (Pospelove) [reply overlay_ports/chakracore/add-missing-reference.patch:11]: it's race condition fix (vcpkg/master)
see https://github.com/microsoft/vcpkg/pull/27968

**Comment by Pospelove** [comment]
@nic11 
The PR to Microsoft/vcpkg has been approved. They think that port changes overall are correct.

Please feel free to push to this branch, since as far as I know we need to update Docker images as well

**Comment by nic11** [comment]
Right, almost forgot about them :D

Actually I think it makes sense to just update vcpkg dropping this port once they merge it. And later make this overlay_port again if necessary to check if everything is fine on our side while we wait for upstream approval (or if they don't merge for a long time)

**Comment by Pospelove** [comment]
@nic11
I use such PRs for testing. vcpkg CI only checks for successful building. I wanted to check if the real project builds with chakra port onboard

**Comment by Pospelove** [comment]
@nic11 
Could you please review this PR?

Approve + merge OR close OR request changes

**Comment by nic11** [comment]
@Pospelove ok then, I'll close it for now. Since it got merged to the upstream, I'm going to update vcpkg, delete port, and rebuild image soon.

**Pull request closed without merging**